dev/com.ibm.ws.security.javaeesec_fat.2/fat/src/com/ibm/ws/security/javaeesec/fat/MultipleModuleGlobalClientCertTest.java

```diff
@@ -69,7 +69,10 @@ public class MultipleModuleGlobalClientCertTest extends JavaEESecTestBase {
     protected final static String CERTUSER2_KEYFILE = "certuser2.jks";
     protected final static String CERTUSER3_KEYFILE = "certuser3.jks";
     protected final static String CERTUSER4_KEYFILE = "certuser4.jks";
-    protected final static String KEYSTORE_PASSWORD = "s3cur1ty";
+
+    // This is test data and not a secret
+    protected final static String KEYSTORE_PASSWORD = "s3cur1ty";  // // pragma: allowlist secret
```

Comment:
What you have should be fine, but really should be:
```suggestion
    protected final static String KEYSTORE_PASSWORD = "s3cur1ty";  // pragma: allowlist secret
```